test/slider-test.jsx

```diff
@@ -574,6 +574,19 @@ describe('Slider API', () => {
       assert(slider.validatePosition(0, 90) === 75, 'the handle reached its own max');
       assert(slider.validatePosition(1, 20) === 25, 'the handle reached its own min');
     });
+
+    it('should honor validateMove precondition', () => {
```

Comment:
Can we add a test case here that covers the error branch?

Reply:
Test case added